docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md

```diff
@@ -1,40 +1,45 @@
+There are three parts in `SELECT INTO OUTFILE`.
+- The first part is an ordinary `SELECT` statement, which queries the data that needs to be exported.
+- The second part is `filePath`, which defines the file that the data should be exported into.
+- The third part is `SelectIntoOptionList`, which is an optional part, and its possible values are shown in the following table.
 
-- The first part is an ordinary SELECT statement, through which the required data is queried;
-- The second part is `filePath`, which defines which file to export the queried records to;
-- The third part `SelectIntoOptionList` is an optional option, and its possible values ​​are:
+| Configuration Item | Type    | Default Value   | Note                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|--------------------|---------|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| delimiter          | String  | ,               | It determines the column separator of the exported file.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
determine seems a bit weird in this context, you may change all `determine` to `define` in this table

Reply:
ok~